pkg/plugins/connection/ssh.go

```diff
@@ -32,7 +32,13 @@ func (s SSH) String() string {
 
 // SSHOpen is a Connection plugin that opens a connection with a device
 type SSHOpen struct {
-	Meta *gornir.TaskMetadata // Task metadata
+	Meta           *gornir.TaskMetadata                                         // Task metadata
+	ClientConfigFn func(*gornir.Host, gornir.Logger) (*ssh.ClientConfig, error) // SSH client configuration
+}
+
+// UserAuthMethods allow the user to define their own SSH auth methods
```

Comment:
is this a leftover? don't see this interface being used anywhere

Reply:
yes, thanks for catching it!